projects/event-lambdas/src/event-api-lambda/application.ts

```diff
@@ -65,5 +68,53 @@ export const createApp = (initConfig: AppConfig): express.Application => {
       )
   );
 
+    app.get(
+        "/tracking-pixel",
+        async (req: Request, res: Response) =>
+            authenticatePandaUser(
+                initConfig.panDomainAuthentication,
+                req,
+                res,
+                async ({ email }: User) => {
+                    const {app, stage} = req.query;
+                    const {hostname, pathname} = url.parse(req.header("referrer") || "");
```

Comment:
Pathname won't work here as it's cross domain. We could either change the referrer policy or provide it as a query param.

Reply:
I've updated this to accept path as a query param.